deepcell/model_zoo/featurenet.py

```diff
@@ -313,7 +314,10 @@ def bn_feature_net_3D(receptive_field=61,
                       padding=False,
                       padding_mode='reflect',
                       multires=False,
-                      include_top=True):
+                      include_top=True,
+                      temporal=None,
```

Comment:
validate this?  If someone uses `"LSTM"` or `"lstm`, should they do the same thing?